Source/WTF/wtf/MemoryPressureHandler.cpp

```diff
@@ -484,8 +493,8 @@ void MemoryPressureHandler::setDispatchQueue(OSObjectPtr<dispatch_queue_t>&& que
 MemoryPressureHandler::Configuration::Configuration()
     : baseThreshold(std::min(3 * GB, ramSize()))
     , baseThresholdVideo(1 * GB)
-    , conservativeThresholdFraction(s_conservativeThresholdFraction)
-    , strictThresholdFraction(s_strictThresholdFraction)
+    , conservativeThresholdFraction(from_env_or_default("WPE_MEMORY_PRESSURE_HANDLER_CONSERVATIVE_THRESHOLD_FRACTION",s_conservativeThresholdFraction))
+    , strictThresholdFraction(from_env_or_default("WPE_MEMORY_PRESSURE_HANDLER_STRICT_THRESHOLD_FRACTION", s_strictThresholdFraction))
     , killThresholdFraction(s_killThresholdFraction)
```

Comment:
You can set the conservative and strict fractions through the MemoryPressureHandlerConfiguration API. These new env vars are not needed, please use the API

Reply:
sure - will remove this part

Reply:
As stated by @magomez , you can make changes around these settings from rdkservices:
https://github.com/rdkcentral/rdkservices/blob/sprint/24Q4/WebKitBrowser/WebKitImplementation.cpp#L500